recipes/CommonVoice/common_voice_prepare.py

```diff
@@ -15,9 +15,22 @@
 import torchaudio
 import unicodedata
 from tqdm.contrib import tzip
+import shutil
+
+import requests
+from tqdm import tqdm
+import tarfile
+
 
 logger = logging.getLogger(__name__)
 
+_URL_TEMPLATE = (
```

Comment:
Where did you find the link? Is it an official link provided by common voice?

Reply:
No There's no direct download link on the official website. I could either download it this way or using dataset lib.

Reply:
OK. So, I would like you to remove everything concerning this new feature (just come back to the initial version). I know it is easier to download everything like that but we can't introduce non-official link. We have no guarantee on the QoS of this link and the content downloaded.... So please, come back to the initial version

Reply:
done